docs/snippets/install-kn.md

```diff
@@ -1,17 +1,19 @@
 !!! todo "Installing the `kn` CLI"
 
     === "Using Homebrew"
-        For macOS, you can install `kn` by using <a href="https://github.com/knative/homebrew-client" target="_blank">Homebrew</a>.
+        For macOS, you can install `kn` by using [Homebrew](https://brew.sh"){target=_blank}.
```

Comment:
Need to delete the quote after `brew.sh`
/lgtm cancel

Reply:
😄 I just copy the file from the one you created https://github.com/knative/docs/blob/mkdocs/docs/snippets/install-kn.md
I guess we need to do another PR to fix in mkdocs branch